imageroot/actions/set-route/validate-input.json

```
@@ -71,7 +71,7 @@
     "properties": {
         "instance": {
             "type": "string",
-            "pattern": "^[a-zA-Z0-9_-]+$",
+            "pattern": "^[a-zA-Z0-9_\\.-]+$",
```

The regex will also match the `\` char https://regex101.com/r/7qo39U/1
```suggestion
            "pattern": "^[a-zA-Z0-9_\.-]+$",
```

---

I test it

---

Sorry to bother @Amygos , it fails

```
JSON Schema input validation aborted at step /home/traefik1/.config/actions/set-route/validate-input.json: Schema compile error for file /home/traefik1/.config/actions/set-route/validate-input.json: invalid character '.' in string escape code
```

I did test  : `            "pattern": "^[a-zA-Z0-9_\.-]+$",`

---

Did you try to see if the `\' char passed the validation in your code?

---

Ok, maybe there is some JSON escaping in place.

```
[root@rl1 ~]# api-cli run module/traefik1/set-route -d '{"instance": "foo.bar", "url": "http://127.0.0.0:2000", "host": "foo.example.org", "lets_encrypt": true, "http2https": true, "skip_cert_verify": true}'
Warning: using user "cluster" credentials from the environment
""
[root@rl1 ~]# api-cli run module/traefik1/set-route -d '{"instance": "foo\bar", "url": "http://127.0.0.0:2000", "host": "foo.example.org", "lets_encrypt": true, "http2https": true, "skip_cert_verify": true}'
Warning: using user "cluster" credentials from the environment
Validation errors: [instance: Does not match pattern '^[a-zA-Z0-9_\.-]+$']
```